frontend/app_flowy/lib/main.dart

```diff
@@ -16,11 +17,32 @@ class FlowyApp implements EntryPoint {
 }
 
 Future<void> main() async {
```

Comment:
Since the two functions only differ in the hotkey manager, try to use the below function instead.

```dart
Future<void> main() async {
  WidgetsFlutterBinding.ensureInitialized();
  await EasyLocalization.ensureInitialized();

  if (Platform.isAndroid || Platform.isIOS) {
    // Mobile platform is not supported the hotKeyManager
  } else {
    WidgetsFlutterBinding.ensureInitialized();
    await hotKeyManager.unregisterAll();
  }

  await FlowyRunner.run(FlowyApp());
}
```